broker/lib/fabrik/DirectorInstance.js

```diff
@@ -23,28 +23,6 @@ class DirectorInstance extends BaseInstance {
     this.networkSegmentIndex = undefined;
```

Comment:
similar changes not required for docker - https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/broker/lib/fabrik/DockerInstance.js#L36-L58 ?

Reply:
Oops! I made a mistake. Actually this cannot be removed. DashboardController uses it still!